mongodb/driver.go

```diff
@@ -58,6 +59,10 @@ func (m *MongoConnectionConfig) GetConnectionURI(isSSL bool) string {
 }
 
 func Open(m *MongoConnectionConfig) (*MongoConnection, error) {
+	if strconv.IntSize < 64 {
```

Comment:
Rather than inlining the `64 bit` restriction in the conditional, can this be held in a constant in the block of constants above and referenced here?